src/Explorer/Panes/SettingsPane/SettingsPane.tsx

```diff
@@ -792,7 +800,7 @@ export const SettingsPane: FunctionComponent<{ explorer: Explorer }> = ({
               </AccordionPanel>
             </AccordionItem>
           )}
-          {shouldShowParallelismOption && (
+          {shouldShowParallelismOption && configContext.platform !== Platform.Emulator && (
```

Comment:
For some of these settings, could we just add the Emulator check to the shouldShow* variables starting on line 177 ?